lib/dolly/property_manager.rb

```diff
@@ -43,5 +44,10 @@ def assign_identity_properties(opts = {})
       id_presence = opts[:id] || opts[:_id] || opts['id'] || opts['_id']
       self.id = id_presence if id_presence
     end
+
+    def assign_rev_properties(opts = {})
+      rev_presence = opts[:rev] || opts [:_rev] || opts['rev'] || opts['_rev']
```

Comment:
maybe: `rev_presence = %i[rev _rev].any? { |rev_key| opts.symbolize_keys[rev_key] }` or `rev_presence = ['rev', '_rev', :_rev, :rev].any? {|key| opts[key] }` 🤔

Reply:
i was following the previous convention at https://github.com/amco/dolly/pull/160/files#diff-9746eb031150d69089faf5a2d17b17c3b15f36d1b48c9e416dd6465e6805809bR44, i'll change both if you think is worth it

Reply:
ok, leave it as it is, no change